packages/p2p/src/components/table/infinite-loader-list.jsx

```diff
@@ -43,16 +44,20 @@ export const InfiniteLoaderList = ({
             loadMoreItems={loadMore}
         >
             {({ onItemsRendered, ref }) => (
-                <List
-                    height={height || 452}
-                    width={width || 960}
-                    itemCount={item_count}
-                    itemSize={item_size || 56}
-                    onItemsRendered={onItemsRendered}
-                    ref={ref}
-                >
-                    {RowRenderer}
-                </List>
+                <AutoSizer style={{ height: (initial_height || 600) }}>
```

Comment:
I think we can remove `<TableDimensions />` since `<Autosizer />` is the same thing?

Reply:
Didn't mean to approve it misclick.

Reply:
@oskar-binary @mustofa-binary I suggest we check and see why `TableDimensions` can't do it itself so we can avoid adding `react-virtualized-auto-sizer` if we don't get anything else out of it.

Reply:
@mahdipourziaei-binary @oskar-binary as discussed, we will use auto-sizer